sean/class1-1/test.py

```diff
@@ -1 +1 @@
-print("hello")
+print("hello edit")
```

Comment:
제 생각에는 그래도
```suggestion
print("hello edited")
```
가 낫지 싶어요